glustercli/cmd/utils.go

```diff
@@ -57,6 +58,35 @@ func sizeToMb(value string) (uint64, error) {
 	return size, nil
 }
 
+// logn is used to find the unit size the given MB belongs to.
+// 1024 MB will return 1
+// 1048576 Mb will return 2 and so on
+func logn(n, b float64) float64 {
+	return math.Log(n) / math.Log(b)
+}
+
+// HumanReadable converts the MB gives into respective unit which is readable
+//
+// HumanReadable(1024) returns 1.0 GB
+// HumanReadable(1536) returns 1.5 GB
+// HumanReadable(1048576) returns 1.0 TB
+func HumanReadable(value uint64) string {
```

Comment:
It'll be great if you can write unit test for this function.

Reply:
done